module/Application/src/Application/Controller/V1/SessionRestController.php

```diff
@@ -88,7 +88,7 @@ public function create($data)
         if ($sessionOpenResponse['status'] == 201) {
             $this->securityLogger->loginSuccessful($sessionOpenResponse['body']['userId']);
         } elseif ($sessionOpenResponse['status'] == 401) {
-            $this->securityLogger->loginFailed($sessionOpenResponse['body']['error']);
+            $this->securityLogger->loginFailed($sessionOpenResponse['body']['error'], $sessionOpenResponse['body']['userId'] ?? null);
```

Comment:
Sorry, should've mentioned the same stripping needs to be done here 🤦🏽

Reply:
ugh, my bad, I forgot there were 2